src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/service/staff/GetOffenderManagerService.kt

```diff
@@ -76,30 +74,30 @@ class GetOffenderManagerService(
     return overview
   }
 
-  fun getOverview(staffIdentifier: StaffIdentifier): OffenderManagerOverview? =
-    communityApiClient.getStaffByCode(staffIdentifier.staffCode).map { staff ->
-      val overview = findOffenderManagerOverview(staffIdentifier, staff.grade)
-      if (overview.hasWorkload) {
-        overview.nextReductionChange = getReductionService.findNextReductionChange(staffIdentifier)
-        overview.reductionHours = getReductionService.findReductionHours(staffIdentifier)
-        overview.contractedHours = getWeeklyHours.findWeeklyHours(staffIdentifier, staff.grade)
-        offenderManagerRepository.findByCaseloadTotals(overview.workloadOwnerId).let { totals ->
-          overview.tierCaseTotals = totals.map { total -> TierCaseTotals(total.getATotal(), total.getBTotal(), total.getCTotal(), total.getDTotal(), total.untiered) }
-            .fold(TierCaseTotals(BigDecimal.ZERO, BigDecimal.ZERO, BigDecimal.ZERO, BigDecimal.ZERO, BigDecimal.ZERO)) { first, second -> TierCaseTotals(first.A.add(second.A), first.B.add(second.B), first.C.add(second.C), first.D.add(second.D), first.untiered.add(second.untiered)) }
-        }
-        offenderManagerRepository.findCasesByTeamCodeAndStaffCode(staffIdentifier.staffCode, staffIdentifier.teamCode).let { cases ->
-          val crns = cases.map { case -> case.crn }
-          overview.caseEndDue = getSentenceService.getCasesDueToEndCount(crns)
-          overview.releasesDue = getSentenceService.getCasesDueToBeReleases(crns)
+  fun getOverview(staffIdentifier: StaffIdentifier): OffenderManagerOverview? {
```

Comment:
this might be a follow on PR, could we change the return type to be https://github.com/ministryofjustice/hmpps-workload/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/domain/OffenderManagerOverview.kt and map directly to that instead of using this overview object then mapping to it?